Marlin/src/lcd/e3v2/proui/dwin_defines.h

```diff
@@ -87,6 +79,45 @@
 #define DEF_HOTENDPIDT TERN(PREHEAT_1_TEMP_BED, PREHEAT_1_TEMP_HOTEND, 195)
 #define DEF_BEDPIDT TERN(PREHEAT_1_TEMP_BED, PREHEAT_1_TEMP_HOTEND, 60)
```

Comment:
`#define DEF_HOTENDPIDT TERN(PREHEAT_1_TEMP_BED, PREHEAT_1_TEMP_HOTEND, 195)`
`#define DEF_BEDPIDT TERN(PREHEAT_1_TEMP_BED, PREHEAT_1_TEMP_HOTEND, 60)`

Un-related to this PR but these don't work, apart from the obvious typos, PREHEAT_1_TEMP_BED is not used as a boolean, results will always be the int values.

Reply:
you're right, I forgot about this, how it was fixed in my own repo. thanks for the reminder